app/templates/base.html

```diff
@@ -20,6 +20,15 @@
     <script nonce="{{ csp_nonce() }}">
         gtm_anon_id = '{{ gtm_anon_id }}';
     </script>
+
+    <script nonce="{{ csp_nonce() }}">
+      window.sessionData = {{ {
+        "category_code": session.get('category', {}).get('code'),
+        "category_name": (session.get('category_answers', []) | first).get('answer_label') if session.get('category_answers') else None,
+        "category_traversal": session.get('category_answers', []) | map(attribute='answer_label') | list
+      } | tojson | safe }};
```

Comment:
Can these be saved as strings in the array using quotes other than transversal? would be good to understand the output